package.json

```diff
@@ -191,7 +191,6 @@
     "jsdom": "^17.0.0",
     "jsdom-global": "^3.0.2",
     "karma": "^6.4.3",
-    "karma-browserstack-launcher": "^1.4.0",
     "karma-chrome-launcher": "^2.0.0",
```

Comment:
Commit 31fccbf215d2 added the browserstack-support. I think `karma-chrome-launcher`, `karma-detect-browsers`, `karma-ie-launcher`, `karma-opera-launcher` and `karma-safari-launcher` becomes unused if removing the saucelabs support, so they should be removed.

Reply:
some of those are still used in karma-local and will revisit those as part of testing initiative